Sources/Sentry/SentryFileManager.m

```diff
@@ -54,14 +53,14 @@ - (_Nullable instancetype)initWithDsn:(SentryDsn *)dsn
         self.lastInForegroundFilePath =
             [self.sentryPath stringByAppendingPathComponent:@"lastInForeground.timestamp"];
 
+        // Remove old cached events for versions before 6.0.0
```

Comment:
Maybe a tracking issue or something of a reminder to get rid of this in some months or in a v7 or something?
Would be nice to spare this I/O eventually

Reply:
I added a task to Asana.